lib/ukofw/Makefile.uk

```diff
@@ -0,0 +1,8 @@
+$(eval $(call addplatlib_s,kvm,libukofw,$(CONFIG_LIBUKOFW)))
+
+LIBUKOFW_CINCLUDES-y         += -I$(LIBUKOFW_BASE)/include
+LIBUKOFW_CINCLUDES-y         += -I$(CONFIG_UK_BASE)/lib/fdt/include
```

Comment:
This line is not required, as `lib/fdt` makes its headers publicly available (and hence to this library) via the `CINCLUDES` directives in it's `Makefile.uk`. If you experience compile errors you need to include the necessary headers where these are missing (eg `libfdt.h` in `include/uk/fdt.h`).